cli/batch-score-rest.sh

```diff
@@ -268,14 +268,17 @@ response=$(curl --location --request POST $SCORING_URI \
 --header "Content-Type: application/json" \
 --data-raw "{
     \"properties\": {
-        \"dataset\": {
-            \"dataInputType\": \"DatasetVersion\",
-            \"datasetName\": \"$DATASET_NAME\",
-            \"datasetVersion\": \"$DATASET_VERSION\"
+        \"InputData\": {
+            \"mnistinput\": {
+                \"JobInputType\": \"UriFolder\",
+                \"Uri\": \"azureml://locations/$LOCATION/workspaces/$WORKSPACE/data/$DATASET_NAME/versions/labels/latest\"
+            }
         },
-        \"outputDataset\": {
-            \"datastoreId\": \"/subscriptions/$SUBSCRIPTION_ID/resourceGroups/$RESOURCE_GROUP/providers/Microsoft.MachineLearningServices/workspaces/$WORKSPACE/datastores/workspaceblobstore\",
-            \"path\": \"$ENDPOINT_NAME\"
+        \"OutputData\": {
+            \"customOutput\": {
+                \"JobOutputType\": \"UriFolder\",
```

Comment:
Looks like we only support Uri file job output.

Reply:
Actually it's the opposite, it accepts UriFolder even when a file  is provided. I tested it out and it looks to work in this way. Should it be a bug?

Reply:
I have just updated the PR to use a folder instead of a file so we don't enter in complex terrain (based on this week conversations), but we probably need to file a bug around this because UriFile or UriFolder is ignored.